### PR TITLE
[Enhancement](array-type) enable_array_type flag update

### PR DESCRIPTION
**Pull request by cambyzju**
# Proposed changes

Issue Number: close #xxx

## Problem summary

At the first time, we add enable_array_type flag as a session variable inside FE(https://github.com/apache/doris/pull/9921).
If users want use ARRAY, they need add a SQL (set enable_array_type=true) to open ARRAY.

But some release versions later, we may open enable_array_type as default, even we remove these codes, then users have to remove the SQL(set enable_array_type=true). It is not friendly to users.

So now we update the flag enable_array_type into fe.conf.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/test/java/org/apache/doris/analysis/ColumnDefTest.java [hunk 1]
```diff
@@ -133,7 +134,7 @@ public void testBooleanDefaultValue() throws AnalysisException {
 
     @Test
     public void testArray() throws AnalysisException {
-        ctx.getSessionVariable().setEnableArrayType(true);
+        Config.enable_array_type = true;
```
Comment (spaces-X) [inline fe/fe-core/src/test/java/org/apache/doris/analysis/ColumnDefTest.java:137]: MayBe it is more reasonable to put this code in setUp.

Comment (cambyzju) [reply fe/fe-core/src/test/java/org/apache/doris/analysis/ColumnDefTest.java]: done

**Review by spaces-X — APPROVED** [review]
LGTM

**Review by xy720 — APPROVED** [review]
LGTM

**Pull request merged**